modules/custom/movie_controller/movie_controller.info.yml

```diff
@@ -0,0 +1,4 @@
+name: Movie Controller
```

Comment:
Pošto pravimo module, ne bi trebalo da mu je ime 'Movie controller'. Promeni nešto u tipa reservations module, ili movie reservations.